openvdb/openvdb/TypeList.h

```diff
@@ -418,27 +418,27 @@ struct TSRemoveIndicesImpl<TypeList<T, Ts...>, First, Last, Idx>
 ///          a user provided converter which provides a @c Type definition.
 /// @tparam  OpT  User struct to convert each type
 /// @tparam Ts  Remaining types in the @c TypeList
-template<template <typename> class OpT, typename... Ts> struct TSTranformImpl;
+template<template <typename> class OpT, typename... Ts> struct TSTransformImpl;
 
 /// @brief  Partial specialization for an empty @c TypeList
 /// @tparam  OpT  User struct to convert each type
 template<template <typename> class OpT>
-struct TSTranformImpl<OpT> {
+struct TSTransformImpl<OpT> {
```

Comment:
Could we also revert this change? We'd like to review and merge only the changes related to comments in this pass if possible.